doc/api/fs.md

```diff
@@ -959,7 +966,7 @@ The "not recommended" examples above check for existence and then use the
 file; the "recommended" examples are better because they use the file directly
 and handle the error, if any.
 
-In general, check for the existence of a file only if the file won’t be
+In general, check for the existence of a file only if the file won't be
```

Comment:
@refack heh :smile: well there’s gotta be a way to make it work, I’m pretty sure I have seen people write comments in VS with German umlauts ;)